lib/core/reporters/v2.js

```diff
@@ -9,7 +9,26 @@ axe.addReporter(
 			options = {};
 		}
 		var out = helpers.processAggregate(results, options);
+		var orientation =
+			screen.msOrientation ||
+			(screen.orientation || screen.mozOrientation || {});
+
 		callback({
+			toolOptions: options,
```

Comment:
Since we're copy/pasting this frequently, does it belong in its own function?

Then we could do:

```
callback({
   ...getEnvironmentDetails(),
  violations: out.violations,
  passes: out.passes,
  // ...
})
```

I'm not sure whether or not it's "worth it", but I generally consider it time to refactor when the same code is copied into 3 different places.

Reply:
I thought about this, but everything was duplicated across the different reporters so I just went with it. If we did split this out, would it be its own `utils` function?

Reply:
I don't know where it belongs. I don't think it makes sense to put it on the public `axe.utils` namespace.

@WilcoFiers where do "internal" utilities go?